src/main/kotlin/no/nav/familie/ef/sak/api/dto/VurderingDto.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.familie.ef.sak.api.dto
+
+import no.nav.familie.ef.sak.repository.domain.VilkårResultat
+import no.nav.familie.ef.sak.repository.domain.VilkårType
+import java.util.*
+
+data class VurderingDto(val id: UUID,
+                        val behandlingId: UUID,
+                        val resultat: VilkårResultat,
+                        val vilkårType: VilkårType,
```

Comment:
Er det nyttig med tidspunkt for hver vurdering?

Reply:
La til endretAv og endretTid fe21fdc
endretAv gir mailadressen til saksbehandlaren